schemas/user_agent.yml

```diff
@@ -6,8 +6,8 @@
     The user_agent fields normally come from a browser request. They often
     show up in web service logs coming from the parsed user agent string.
   fields:
-    - name: raw
-      type: text
+    - name: original
+      type: keyword
```

Comment:
I would actually make this one multi-field.

Full text search will be useful for weird user agents that aren't recognized by the US parsers (various language HTTP libs, obscure crawlers, etc). Still useful to filter for everything that contains "Python", for example. So for this we need to retain full text search.

So I would go with FTS at the top + `.keyword` for the nested field.

Reply:
Or you can consider this out of scope for now, and wait for the full review of fields in #104 to go MF. In that case we should leave it at `text` indexing.

Reply:
I consider it out of scope for now :-)

Reply:
Agreed. So you'll revert back to `text` indexing?

Reply:
Can do. So far I was thinking it's a bug that it's text :-)